src/main/java/com/mateacademy/service/PersonServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.mateacademy.service;
+
+import com.mateacademy.entity.Person;
+import com.mateacademy.entity.PersonDao;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+/** This class marked as "personService"
+ *  using @Service annotation. Use @Autowired
+ *  annotation to autowire PersonDao bean.
+ *  It contains methods which realize PersonDao
+ *  methods.
+ */
+
+@Service("personService")
+public class PersonServiceImpl implements PersonService {
+
+    @Autowired
+    PersonDao personDao;
```

Comment:
add access modificator
and read about constructor Injection in Spring with lombok https://www.baeldung.com/spring-injection-lombok